From 8f561100e918b1228d236fcb0cfe62eddadea2b7 Mon Sep 17 00:00:00 2001
From: tpaviot <tpaviot@gmail.com>
Date: Thu, 12 Jan 2012 21:41:07 +0100
Subject: [PATCH] [cppcheck-warning-fix][Redundant-assignement]

[Quantity/Quantity_Color.cxx:471]: (warning) Redundant assignment of "R1" in switch
[Quantity/Quantity_Color.cxx:471]: (warning) Redundant assignment of "R2" in switch
[Quantity/Quantity_Color.cxx:471]: (warning) Redundant assignment of "R3" in switch
---
 src/Quantity/Quantity_Color.cxx |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

Index: oce/src/Quantity/Quantity_Color.cxx
===================================================================
--- oce.orig/src/Quantity/Quantity_Color.cxx
+++ oce/src/Quantity/Quantity_Color.cxx
@@ -468,8 +468,7 @@ Standard_ShortReal RB = 0 ;
 switch (AName) {
 
 	case Quantity_NOC_BLACK :
-		R1 = R2 = R3 = 0.0;
-		RR = R1 = 0.000000; RG = R2 = 0.000000; RB = R3 = 0.000000;
+		R1 = R2 = R3 = RR = RG = RB = 0.0;
 	break;
 	case Quantity_NOC_MATRAGRAY :
 		RR = R1 = (float ) 0.600000; RG = R2 = (float ) 0.600000; RB = R3 = (float ) 0.600000;
